phys2denoise/cli/run.py

```diff
@@ -50,11 +63,19 @@ def _get_parser():
         "physiological data, with or without extension.",
         required=True,
     )
+    required.add_argument(
+        "-md",
+        "--mode",
+        dest="mode",
+        type=str,
+        help="Format of the input physiological data. Options are: "
+        "physio or bids. Default is physio.",
+    )
```

Comment:
Let's leave this in for now, but let's also open an issue to remove this option and make a i/o function that automatically recognises the input type based on the given input.

Reply:
It shouldn't be to hard, I can incorporate it for this PR

Reply:
@maestroque where are you regarding that ? If you don't have time for it there is no problem, we can definitely do what Stef suggested ! However, if you decide to do that, can you please add the default value to `physio` ?

Reply:
Please check [this commit](https://github.com/physiopy/physutils/pull/7/commits/1cd7539e3b96042ae51547ad6070b0ee4bdc0531) @smoia I partially implemented it